ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -6,21 +6,8 @@ Feature: Test functionality of integration import export feature
     Given clean application state
     Given "Camilla" logs into the Syndesis
     Given created connections
-      | Twitter | Twitter Listener | Twitter Listener | SyndesisQE Twitter listener account |
-      | Salesforce | QE Salesforce | QE Salesforce | SyndesisQE salesforce test |
-
-
-
-  @integration-import-export-step-clean-application-state
-  Scenario: Clean application state
-    Given clean application state
-    Given "Camilla" logs into the Syndesis
-
-#
-#  1. integration-export
-#
-  @integration-export
-  Scenario: Create integration and test export feature of single integration
+      | Twitter    | Twitter Listener | Twitter Listener | SyndesisQE Twitter listener account |
+      | Salesforce | QE Salesforce    | QE Salesforce    | SyndesisQE salesforce test          |
 
     # create integration
     And "Camilla" navigates to the "Home" page
```

Comment:
And once you have `Given created connections` you can rm some more steps where SF connection is created step by step

Reply:
I am not creating salesforce connection but setting Integration with already created salesforce connection - nothing I can remove, right?